src/icons/glyphs/svgs/SunMoonGlyph.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+function SvgSunMoonGlyph(props: React.SVGProps<SVGSVGElement>) {
+  return (
+    <svg
+      width={48}
+      height={48}
+      viewBox="0 0 48 48"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <mask
```

Comment:
What content should go inside the SVG tags here? I just copied the path from the `sun-moon-glyph.svg` file.

Reply:
This file is auto-generated from `yarn run build:icons:index` so we shouldn't be adding anything here, let me know if anything about the steps are confusing.